qiskit_ibm_runtime/fake_provider/__init__.py

```diff
@@ -33,6 +33,7 @@
 
 .. plot::
```

Comment:
This plot showed three images and I splitting it into three `.. plot::` to be able to set an alt text to each image individually.